fem/transfer.cpp

```diff
@@ -226,12 +226,63 @@ const Operator &InterpolationGridTransfer::BackwardOperator()
    return *B.Ptr();
 }
 
-
 L2ProjectionGridTransfer::L2Projection::L2Projection(
    const FiniteElementSpace &fes_ho_, const FiniteElementSpace &fes_lor_)
    : Operator(fes_lor_.GetVSize(), fes_ho_.GetVSize()),
      fes_ho(fes_ho_),
      fes_lor(fes_lor_)
+{ }
+
+void L2ProjectionGridTransfer::L2Projection::BuildHo2Lor(
```

Comment:
Nice! I like how you structured `L2Projection::BuildHo2Lor` and `L2Projection::ElemMixedMass`.

Reply:
Thanks!